terraform/vars/skylight/main.tf

```diff
@@ -24,3 +24,13 @@ module "network" {
   source = "../../modules/network"
 }
 
+module "fhir" {
+  source         = "../../modules/fhir"
+  region         = "us-west1"
+  time_zone      = "UTC"
+  dataset_name   = "PHDI_DATASET-${time_static.current_date.year}${time_static.current_date.month}${time_static.current_date.day}"
+  fhirstore_name = "PHDI_FHIRSTORE-${time_static.current_date.year}${time_static.current_date.month}${time_static.current_date.day}"
+  fhir_version   = "R4"
+}
+
+resource "time_static" "current_date" {}
```

Comment:
Could we move this into `terraform/modules/fhir/main.tf` and simply append the value of `dataset_name` on line 2? Generally I think it makes more sense to declare resources within modules. Totally open to an argument against this though.

Reply:
Ah... I like that.  Good recommendation!